### PR TITLE
Add script 1 in the `cluster-cell-calling` module

### LINKED ISSUE #87: Add script 1 in the `cluster-cell-calling` module
## What are the scientific goals of the analysis?

To add the `01-cluster-cell-calling.Rmd` script tasked to perform the clustering.

### PR DESCRIPTION
**Pull request by AntoniaChroni**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

To add the `01-cluster-cell-calling.Rmd` script tasked to perform the clustering.

## What GitHub issue does your pull request address?

Closes #87.

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

Please review the code and documentation for clarity.

## Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/cluster-cell-calling/01-cluster-cell-calling.Rmd [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+# Intepretation of clustering trees
+
+We can see that one cluster is very distinct and does not change with the value of \(k\). This is the Center players which are very different to the other positions. On the other side of the tree we see a single cluster that splits into the two clusters we would expect to see. After this the tree becomes messier and there might be even nodes with multiple incoming edges. This is a good indication that we have over clustered the data.
```
Comment (sharonfreshour) [inline analyses/cluster-cell-calling/01-cluster-cell-calling.Rmd:236]: I think this section maybe confusing to include in the report as it is explaining the interpretation of a specific example tree from the `clustree` documentation and the features of the tree being discussed likely won't be applicable across projects/datasets.

**Review by sharonfreshour — APPROVED** [review]
The code sections look logical and most documentation is clear. I did make a suggestion to possibly remove one section from the report, related to interpreting clustering trees.

File: analyses/cluster-cell-calling/01-cluster-cell-calling.Rmd (hunk 1 above)
Comment (AntoniaChroni) [reply analyses/cluster-cell-calling/01-cluster-cell-calling.Rmd]: @sharonfreshour Please let me know how it reads now, see f44916f4a652dccf09802d5d2fa96dc2acfa660a.
I also updated the `README.md` file accordingly, see 923573b30b58ae352f45683a73c673ab7543bbf8.

Comment (sharonfreshour) [reply analyses/cluster-cell-calling/01-cluster-cell-calling.Rmd]: @AntoniaChroni I think the updated version of the report looks good and is more clear now how interpretation will apply to an any dataset. Updates in README are clear as well.

**Pull request merged**